.gitignore

```diff
@@ -8,3 +8,4 @@
 /bin/
 /composer.phar
 /nbproject
+/.idea
```

Comment:
i using a phpStorm not NetBeans. Some people who use Netbeans after next push will push "nbproject" directory to repo if i remove it. You sure want to remove nbproject ?